pydy/codegen/ode_function_generators.py

```diff
@@ -417,7 +430,7 @@ def _convert_specifieds_dict_to_array(self, x, t, r):
             # TODO : Not sure if this is the best check here.
             if isinstance(type(k), UndefinedFunction):
                 k = (k,)
-            idx = [self.specifieds.index(symmy) for symmy in k]
+            idx = [list(self.specifieds).index(symmy) for symmy in k]
```

Comment:
The `define_inputs()` method is currently a public method and in the base class it produces a certain type, i.e. list of sequences. But this line is in the subclass. I'd prefer the super method and the method in this class to return the same type.